packages/babel-traverse/test/traverse.js

```diff
@@ -1,7 +1,7 @@
-let traverse = require("../lib").default;
-let assert   = require("assert");
-let _        = require("lodash");
-let parse = require("babylon").parse;
+let cloneDeep = require("lodash/cloneDeep");
+let traverse  = require("../lib").default;
+let assert    = require("assert");
+let parse     = require("babylon").parse;
```

Comment:
I also could've used arrow functions for the `forEach`'s but I didn't because that's not what this PR is about.